templates/css_declarations.erb

```diff
@@ -0,0 +1,3 @@
+<% @css_sheets.each do |sheet| -%>
```

Comment:
@css_sheets.select { |x| x['local'] }.each ...

It would also be more correct to do url escaping followed by html escaping. This can be done with `CGI::escape` and `CGI::escapeHTML` in Ruby.